app/src/main/java/com/github/multimatum_team/multimatum/DeadlineDetailsActivity.kt

```diff
@@ -43,10 +36,18 @@ class DeadlineDetailsActivity : AppCompatActivity() {
         // Set the texts for the title and the date of the deadline
         findViewById<TextView>(R.id.deadline_details_activity_title).text = title
         findViewById<TextView>(R.id.deadline_details_activity_date).text =
-            getString(R.string.DueTheX, date)
+            getString(R.string.DueTheXatX, date.toLocalDate(), date.toLocalTime())
 
         // Set the detail text to inform the user if it is due, done or the remaining time
         val detailView = findViewById<TextView>(R.id.deadline_details_activity_done_or_due)
+        setDetailText(detailView, state, date)
+
+    }
+
+    /*
+     Set the detail text to inform the user if it is due, done or the remaining time
+     */
```

Comment:
It's the wrong syntax for comment, but since. Even if it's a private function, try to use "/** ... */" instead (it then appears in green in android studio)